.github/workflows/increment-plugin-versions.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: Increment Plugin's Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
```

Comment:
Do we need log level configuration in this workflow? Can we delete this?

Reply:
We need this @dblock for 1st time once we kick of the workflow, the intention is to run in debug mode so that we can notice the exact errors, once confident, I will create a PR to run as cron, remove entire `workflow_dispatch`.